Doc/whatsnew/3.12.rst

```diff
@@ -523,6 +523,12 @@ Pending Removal in Python 3.14
   functions that have been deprecated since Python 2 but only gained a
   proper :exc:`DeprecationWarning` in 3.12. Remove them in 3.14.
 
+* Accessing ``co_lnotab`` was deprecated in :pep:`626` since 3.10
+  and was planned to be removed in 3.12
+  but it only got a proper :exc:`DeprecationWarning` in 3.12.
+  Remove it in 3.14.
```

Comment:
We don't have to remove it in 3.14.
Personally I think we should deprecate early and remove late (although I failed to deprecate early in this case 🙂)

Perhaps "May be removed in 3.14"?

Reply:
Yes, your wording is better, because it clearly gives us more freedom 👍